external-docs/docs/guides/configuration/qube-config/columns/attributes/attribute-resources.md

```diff
@@ -0,0 +1,199 @@
+* the `required` field defaulting to `false` - this means that csvcubed will accept a CSV file with empty cells in the
+`Status` column;
+* [attribute values](./attribute-values.md) automatically being generated from the unique cell values in the `Status` column (`Provisional` and `Final` in this example).
+
+If all cells in the `Status` column should be populated, set the `required` field to `true`. This will prompt csvcubed to flag to the user if there are any blank cells in the `Status` column:
+
+```json
+{
+   "$schema": "https://purl.org/csv-cubed/qube-config/v1",
+   "title": "The title of the cube",
+   "description": "A description of the contents of the cube",
+   "columns": {
+      "Status": {
+         "type": "attribute",
+         "required": true
+      }
+   }
+}
+```
+
+## Label, description and definition
+
+Additional details can be associated with the attributes in your data set through the `label`, `description` and
+`definition_uri` fields.
+
+As mentioned above, the `label` field will default to the column title unless explicitly configured in the
+`qube-config.json` file. In the example below, the `Status` label is amended to `Observation status`:
+
+```json
+{ ...
+   "columns": {
+      "Status": {
+         "type": "attribute",
+         "label": "Observation status"
+      }
+   }
+}
+```
+
+The `description` field can be used to provide a longer description of your attribute. If you want to provide
+information about your methodology, the `description` field is the preferred place for this. It also supports the
+markdown format.
+
+```json
+{ ...
+   "columns": {
+      "Status": {
+         "type": "attribute",
+         "description": "The status of the observed value (Provisional or Final)"
+      }
+   }
+}
+```
+
+The `definition_uri` fields allows you to refer to external resources that further define an attribute's values:
+
+```json
+{ ...
+   "columns": {
+      "Status": {
+         "type": "attribute",
+         "definition_uri": "https://sdmx.org/wp-content/uploads/01_sdmx_cog_annex_1_cdc_2009.pdf"
+      }
+   }
+}
+```
+
+## Attribute values
+
+Rather than allowing csvcubed to automatically generate attribute values from the unique cell values in the `Status` column, the `values` field can be configured to specify a list of permitted cell values:
+
+```json
+{ ...
+   "columns": {
+      "Status": {
+         "type": "attribute",
+         "values": [
+            {
+               "label": "Provisional"
+            },
+            {
+               "label": "Final"
+            }
+         ]
+      }
+   }
+}
+```
+
+See the [Attribute values](./attribute-values.md) page for more information on the configuration options available for the `values` field.
+
+## Inheritance
+
+To reuse or extend an existing attribute, the `from_existing` field can be configured to link to a URI where the
+attribute to be reused or extended is defined.
```

Comment:
A URI doesn't need to be a link to anywhere that defines anything. It's just an identifier.